zqd/search/pcapsearch.go

```diff
@@ -0,0 +1,69 @@
+package search
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/brimsec/zq/pcap"
+	"github.com/brimsec/zq/pcap/pcapio"
+	"github.com/brimsec/zq/pkg/fs"
+	"github.com/brimsec/zq/zqd/api"
+)
+
+type PcapSpace interface {
+	PcapIndexPath() string
+	PcapPath() string
+}
```

Comment:
It would be great to consolidate these interfaces in one location- it would make this clearer for me. That said I'm not quite sure where that one location should be. A lot is changing around this I suppose it would be best to wait and see how this shakes out.

Reply:
Yes - in another comment, I mentioned that we should be able to move the pcap handling functionality off of the space & onto the store.